test/itemlist_formaction.cpp

```diff
@@ -12,6 +12,7 @@ using namespace newsbeuter;
 
 
 TEST_CASE("process_op(OP_OPEN)", "[itemlist_formaction]") {
+	INFO("Opens an article and display its attribute. This is achieved by setting the pager command to echo the article attributes to a temporary file and parsing it.")
```

Comment:
Thanks for the details, that definitely makes sense, I'll try to respect that in my next modifications.